Bottom: e7d8d4df7cfec3586065102aa061a60be0706578
Top:    80d8d6c32238302049f42b9b7d74d6e43692bd55
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-07-05 16:58:16 +0800

Refresh of virtio-net-rename-virtnet_xdp

---

diff --git a/drivers/net/virtio_net.c b/drivers/net/virtio_net.c
index 9a9845c..ece9784 100644
--- a/drivers/net/virtio_net.c
+++ b/drivers/net/virtio_net.c
@@ -2346,10 +2346,10 @@ static int virtnet_bpf(struct net_device *dev, struct netdev_bpf *bpf)
 {
 	switch (bpf->command) {
 	case XDP_SETUP_PROG:
-		return virtnet_xdp_set(dev, xdp->prog, xdp->extack);
+		return virtnet_xdp_set(dev, bpf->prog, bpf->extack);
 	case XDP_QUERY_PROG:
-		xdp->prog_id = virtnet_xdp_query(dev);
-		xdp->prog_attached = !!xdp->prog_id;
+		bpf->prog_id = virtnet_xdp_query(dev);
+		bpf->prog_attached = !!bpf->prog_id;
 		return 0;
 	default:
 		return -EINVAL;
